docs/develop/built-in-ui-ids.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Find the IDs of built-in Office ribbon tabs
+description: Find the IDs of the built-in Office ribbon tabs.
+ms.date: 07/31/2024
+ms.topic: reference
+ms.localizationpriority: medium
+---
+
+<!-- 
+  This article is deliberately left out of the Office Add-ins TOC because 
+  it will be moving over to the M365 doc set as soon as that is up and running. 
+-->
+
+# Find the IDs of built-in Office ribbon tabs
+
+The following table shows the valid built-in Office ribbon tab `id` values by application. The columns indicate on which platform the IDs are supported. For example, **TabHome** is supported in Word 2016 or later on Windows and Word on the web, but TabBlogPost is supported only in Word desktop.
+
+| Office application | Supported on desktop, on the web,</br>and new Outlook on Windows (preview) | Supported only on desktop |
```

Comment:
```suggestion
| Office application | Supported on desktop, on the web,</br>and new Outlook on Windows | Supported only on desktop |
```
We're in the process of removing preview strings from new Outlook on Windows references.